examples/forms.amp.html

```diff
@@ -535,5 +535,18 @@ <h4 id="header1">Using button[type=submit]</h4>
 </form>
 
 
+<h4 id="header1">Redirect After POST (works only if running on localhost)</h4>
+<form method="post"
+      action-xhr="http://iframe.localhost:8000/form/redirect-to/post"
+      target="_blank">
```

Comment:
> Lets make redirect on _blank an error for now and add support for it in a follow up PR

This will require us to move the logic to handling redirect to `amp-form` instead of `xhr` since xhr has no context of where the redirect should happen.